application.go

```diff
@@ -149,6 +150,13 @@ type Stats struct {
 	LifeTime map[string]float64 `json:"lifeTime"`
 }
 
+// Secret is a reference to an existing secret object whose value may be used
```

Comment:
I don't understand this comment like at all :/ Can you be more precise what the "value" is here?